docs/en/advanced_guides/transforms.md

```diff
@@ -1 +1,59 @@
 # Transforms
+
+- [Transforms]()
+  - [Overview of transforms](#overview-of-transforms)
+  - [Introduction of `MultiView`](#introduction-of-multiview)
+  - [Introduction of `PackSelfSupInputs`](#introduction-of-packselfsupinputs)
+
+
+## Overview of transforms
+We have introduced how to build a `Pipeline` in [add_transforms](./add_transforms.md). A `Pipeline` contains a series of
+`transforms`. There are three main categories of `transforms` in MMSelfSup:
+1. Transforms about processing the data. The unique transforms in MMSelfSup are defined in [processing.py](../../../mmselfsup/datasets/transforms/processing.py), e.g. `RandomCrop`, `RandomResizedCrop` and `RandomGaussianBlur`.
```

Comment:
```suggestion
1. Transforms about processing the data. The unique transforms in MMSelfSup are defined in [processing.py](https://github.com/open-mmlab/mmselfsup/blob/1.x/mmselfsup/datasets/transforms/processing.py), e.g. `RandomCrop`, `RandomResizedCrop` and `RandomGaussianBlur`.
```

Reply:
use github link instead of relative path

Reply:
done